exec_01.ts

```diff
@@ -0,0 +1,35 @@
+//Exercicio 01 - A
+function contadorVogais(nome: string) {
+    var nome = nome.toLowerCase()
```

Comment:
O var é um jeito antigo de declarar variáveis em JS que não é mais utilizado hoje em dia, a não ser talvez em casos muito específicos. (https://hcode.com.br/blog/entenda-a-diferenca-entre-var-let-e-const-no-javascript)

Então o que for variável deve ser declarado com 'let' e o que for constante com 'const'.

'nome' aqui é uma constante.

No arquivo de js, você utilizou ; ao final das linhas, é sempre bom manter o mesmo padrão pro projeto inteiro. Eu particularmente recomendo sempre usar o ; em JS/TS, mesmo não sendo obrigatório.

Reply:
Corrigi colocando o "nome.toLowerCase()" direto na antiga linha 5.

-> const letras: Array<string> = nome.toLowerCase().split('')

Reply:
Sobre o " ; " não estou usando no arquivo.ts e quando faço o comando "npx tsc nomedoarquivo.ts" para criar o arquivo.js automaticamente aparece os " ; ".